circle.yml

```diff
@@ -12,6 +12,6 @@ test:
     - make test
     - nvm install 0.12.7
     - nvm alias default 0.12.7
```

Comment:
Unfortunately not, CircleCI doesn't have 0.12.7 installed by default for some reason. If you do `nvm use 0.12` it will run against 0.12.0.